### ISSUE TITLE
Results with RandAug?

### ISSUE THREAD

**Issue by ShijianXu** [kentaroy47/vision-transformers-cifar10#18]
Hello,
Do you have the result for SimpleViT? Also, may I ask do you use random augmentation for the provided results? How much does the random augmentation affect the final results?
Thank you.

**Comment by kentaroy47** [comment]
I mostly use random augmentation for the reported results.
Without it, it will overfit. 

I think SimpleViT was similar to ViT-s (90%?)

**Comment by ShijianXu** [comment]
I see. Thank you for your reply. I didn't use random augmentaion. That's might explain why my validation loss overfit quickly and the test accuracy not the high.
